tests/func/test_get.py

```diff
@@ -179,3 +180,15 @@ def test_get_from_non_dvc_master(tmp_dir, erepo_dir, caplog):
 
     assert caplog.text == ""
     assert (tmp_dir / dst).read_text() == "some_contents"
+
+
+def test_get_url(tmp_dir, erepo_dir):
+    with erepo_dir.chdir():
+        erepo_dir.dvc_gen("foo", "foo")
+        assert main(["get", ".", "foo", "--show-url"]) == 0
```

Comment:
If we use `erepo` I would recommend making assert outside of `chdir` and replacing "." with fspath(erepo_dir). If we want to test in the same dir, lets just use `dvc` fixture :)

Reply:
and by making assert outside, I just mean removing the intdentation

Reply:
Consider using caplog fixture and checking whether expected log message(s) is(are) in place

Reply:
thanks, that's better. will change.